packages/core/src/modules/attendance/__test__/attendance-service.e2e-spec.ts

```diff
@@ -3,17 +3,16 @@ import { AttendancePoolWrite, AttendanceWrite, AttendeeWrite, EventWrite, UserWr
 import crypto from "crypto"
 import { ulid } from "ulid"
 import { afterEach, beforeEach, describe, expect, it } from "vitest"
+import assert from "../../../../assert"
 import { CleanupFunction, createServiceLayerForTesting } from "../../../../vitest-integration.setup"
-import { createServiceLayer, type ServiceLayer } from "../../core"
+import { type ServiceLayer, createServiceLayer } from "../../core"
```

Comment:
Can we get some import sorting into CI?

Reply:
It would be really nice to have biome fix imports, I don't think it does any import checking atm

Reply:
https://linear.app/dotkom/issue/DOT-690/make-biome-sort-and-fix-imports